modules/nodeColorManager/pom.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.gephi</groupId>
+        <artifactId>gephi-plugin-parent</artifactId>
+        <version>0.9.0</version>
+    </parent>
+
+    <groupId>edu.buaa.act</groupId>
+    <artifactId>gephi-plugin-node-color-manager</artifactId>
+    <version>1.0.0</version>
+    <packaging>nbm</packaging>
+
+    <name>Node Color Manager</name>
+
+    <dependencies>
+        <!-- Insert dependencies here -->
+        <dependency>
+            <groupId>org.gephi</groupId>
+            <artifactId>tools-api</artifactId>
+            <version>0.9.0</version>
```

Comment:
That is unrelated. Your plugin finds its dependency to Gephi by inheriting the `gephi-plugin-parent` configuration, which defines explicitly a Gephi version (0.9.0 at the moment). When we update Gephi let's say to 0.9.1 we'll update this configuration and create a `0.9.1` version of the `gephi-plugin-parent`. Plugins will just need to upgrade their dependency setting for `gephi-plugin-parent`. At that time you'll have to update this line as well if you don't remove it.